configure.py

```diff
@@ -125,6 +125,8 @@ def _get_brew_prefix():
 
 
 def _file_chk(path, chksum_hexdigest):
+    if chksum_hexdigest is None:
+        return True
```

Comment:
I'd much prefer an explicit `"skip"` keyword, otherwise it's likely the next time we add an entry, we end up copying one with no checksum and forget to add it.

Also: "useful" typo in commit description